packages/react/src/components/ExpressiveModal/ExpressiveModalCloseBtn.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { Close20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Expressive Modal
+ *
+ * @param {object} props react proptypes
```

Comment:
Can you add JSDocs for the props?

Reply:
Sorry, isn't that what this line is?

Reply:
See example here:
https://github.com/carbon-design-system/ibm-dotcom-library/blob/ebc6b1f4dbf82e0f3eefb8bf53b42bff7992edfe/packages/react/src/components/HorizontalRule/HorizontalRule.js#L31

Reply:
At first, glance that looks inaccurate, but I'll dive in and take a closer look. Thanks for the example!

Reply:
I see it now, thanks! I'll make the update.